src/main/java/org/spongepowered/common/data/builder/manipulator/mutable/item/ItemAuthorDataBuilder.java

```
@@ -0,0 +1,75 @@
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.data.builder.manipulator.mutable.item;
+
+import com.google.common.base.Optional;
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemStack;
+import org.spongepowered.api.data.DataHolder;
+import org.spongepowered.api.data.DataView;
+import org.spongepowered.api.data.key.Keys;
+import org.spongepowered.api.data.manipulator.DataManipulatorBuilder;
+import org.spongepowered.api.data.manipulator.immutable.item.ImmutableAuthorData;
+import org.spongepowered.api.data.manipulator.mutable.item.AuthorData;
+import org.spongepowered.api.service.persistence.InvalidDataException;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.Texts;
+import org.spongepowered.common.data.manipulator.mutable.item.SpongeAuthorData;
+import org.spongepowered.common.data.util.DataUtil;
+import org.spongepowered.common.data.util.NbtDataUtil;
+
+public class ItemAuthorDataBuilder implements DataManipulatorBuilder<AuthorData, ImmutableAuthorData> {
+
+    @Override
+    public Optional<AuthorData> build(DataView container) throws InvalidDataException {
+        DataUtil.checkDataExists(container, Keys.BOOK_AUTHOR.getQuery());
+        final String json = DataUtil.getData(container, Keys.BOOK_AUTHOR, String.class);
+        final Text author = Texts.json().fromUnchecked(json);
```

Should likely surround with try catch and throw an `InvalidDataException` if and when `Text` will throw it's own exceptions.